packages/appeal-reply-service-api/src/controllers/reply.js

```diff
@@ -75,22 +76,28 @@ module.exports = {
       .then(async (originalDoc) => {
         // logger.debug(`Original doc \n${originalDoc.reply}`);
         logger.debug(originalDoc.reply, 'STEVE_API');
-
-        const isFirstSubmission = originalDoc.state !== 'SUBMITTED' && newDoc.state === 'SUBMITTED';
-
-        if (isFirstSubmission) newDoc.submissionDate = new Date().toISOString();
-
         await mongodb
           .get()
           .collection(dbId)
-          .updateOne({ _id: idParam }, { $set: { uuid: idParam, reply: newDoc } })
-          .then(() => {
+          .updateOne({ _id: idParam }, { $set: { uuid: idParam, reply: req.body } })
+          .then(async () => {
             logger.debug(`Updated reply ${idParam}\n`);
 
-            if (isFirstSubmission) {
+            if (originalDoc.state !== 'SUBMITTED' && newDoc.state === 'SUBMITTED') {
               logger.info({ newDoc }, 'STEVE_IS_FIRST_SUBMISSION');
-              queue.addAppealReply(newDoc);
-              // TODO: call to notify will go here
+              await queue.addAppealReply(newDoc);
+            }
+
+            if (req.body.state === 'SUBMITTED') {
+              try {
+                await notify.sendAppealReplySubmissionConfirmationEmailToLpa(req.body);
+                res.status(200).send(req.body);
+              } catch (err) {
+                logger.error(`Problem sending email ${idParam}\n${err}`);
```

Comment:
This will send an email on every update? Or does the state change from 'SUBMITTED' somewhere else?